scripts/config.py

```diff
@@ -52,6 +52,7 @@ def __init__(self):
 
         self.pinecone_api_key = os.getenv("PINECONE_API_KEY")
         self.pinecone_region = os.getenv("PINECONE_ENV")
+        self.pinecone_table = os.getenv("PINECONE_TABLE")
```

Comment:
It would be nice to default to `auto-gpt` to allow overriding, the table is created if it doesn't already exist (perhaps that should be mentioned in the the `README.md`:

```suggestion
        self.pinecone_table = os.getenv("PINECONE_TABLE", "auto-gpt")
```